imaging-ingestion-operator/model/common_test.go

```diff
@@ -0,0 +1,203 @@
+/*
+(C) Copyright IBM Corp. 2021
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package model_test
+
+import (
+	"testing"
+
+	"github.com/Alvearie/imaging-ingestion/imaging-ingestion-operator/model"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// TestMergeEnvsEmptyEmpty calls model.MergeEnvs with a empty slices, checking
```

Comment:
When changing package name to `model`, please remove `model.` from everywhere in this test file.

Reply:
done